rust/chains/hyperlane-ethereum/src/interchain_security_module.rs

```diff
@@ -96,9 +97,9 @@ where
     M: Middleware + 'static,
 {
     #[instrument(err, ret)]
-    async fn module_type(&self) -> ChainResult<u8> {
+    async fn module_type(&self) -> ChainResult<ModuleType> {
         let module_type = self.contract.module_type().call().await?;
-        Ok(module_type)
+        Ok(ModuleType::from_u8(module_type).unwrap_or_default())
```

Comment:
Do we want to log if we encounter an unknown module type? (Maybe you already do elsewhere?)

Reply:
Yeah I think debug or maybe warn